packages/nodes-base/nodes/N8nApi/WorkflowDescription.ts

```diff
@@ -0,0 +1,393 @@
+	{
+		displayName: 'Workflow ID',
+		name: 'workflowId',
+		type: 'string',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: ['workflow'],
+				operation: ['deactivate'],
+			},
+		},
+		description: 'ID of the workflow to deactivate',
+	},
+];
+
+const deleteOperation: INodeProperties[] = [
+	{
+		displayName: 'Workflow ID',
+		name: 'workflowId',
+		type: 'string',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: ['workflow'],
+				operation: ['delete'],
+			},
+		},
+		description: 'ID of the workflow to delete',
+	},
+];
+
+const getAllOperation: INodeProperties[] = [
+	{
+		displayName: 'Return All',
+		description: 'Whether to return all results or only up to a given limit',
+		name: 'returnAll',
+		type: 'boolean',
+		default: true,
+		displayOptions: {
+			show: {
+				resource: ['workflow'],
+				operation: ['getAll'],
+			},
+		},
+	},
+	{
+		displayName: 'Limit',
+		description: 'Max number of results to return',
+		name: 'limit',
+		type: 'number',
+		default: 100,
+		typeOptions: {
+			minValue: 1,
+			maxValue: 250,
+		},
+		displayOptions: {
+			show: {
+				resource: ['workflow'],
+				operation: ['getAll'],
+				returnAll: [false],
+			},
+		},
+		routing: {
+			request: {
+				qs: {
+					limit: '={{ $value }}',
+				},
+			},
+		},
+	},
+	{
+		displayName: 'Additional Fields',
+		name: 'additionalFields',
+		type: 'collection',
+		default: {},
+		displayOptions: {
+			show: {
+				resource: ['workflow'],
+				operation: ['getAll'],
+			},
+		},
+		options: [
+			{
+				displayName: 'Return Only Active Workflows',
+				name: 'activeWorkflows',
+				type: 'boolean',
+				default: false,
+				routing: {
+					request: {
+						qs: {
+							active: '={{ $value }}',
+						},
+					},
+				},
+			},
+			{
+				displayName: 'Tags',
+				name: 'tags',
+				type: 'string',
+				default: '',
+				routing: {
+					// Only include the 'tags' query parameter if it's non-empty
+					send: {
+						type: 'query',
+						property: 'tags',
+						value: '={{ $value !== "" ? $value : undefined }}',
+					},
+				},
+				description: 'Include only workflows with these tags',
+				hint: 'Comma separated list of tags (empty value is ignored)',
+			},
+		],
+	},
+];
+
+const getOperation: INodeProperties[] = [
+	{
+		displayName: 'Workflow ID',
+		name: 'workflowId',
+		type: 'string',
+		default: '1',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: ['workflow'],
+				operation: ['get'],
+			},
+		},
+		description: 'ID of the workflow to get',
+	},
+];
+
+const updateOperation: INodeProperties[] = [
+	{
+		displayName: 'Workflow ID',
+		name: 'workflowId',
+		type: 'string',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: ['workflow'],
+				operation: ['update'],
+			},
+		},
+		description: 'ID of the workflow to update',
+	},
+	{
+		displayName: 'Workflow Object',
+		name: 'workflowObject',
```

Comment:
please add description and placeholder of expected input, create operation always fails for me, same applies for update operation
![image](https://user-images.githubusercontent.com/88898367/189858519-0e7cc1ac-db6c-4157-a40b-7b5a64b659d1.png)

Reply:
Yes, I completely missed it's actually possible to create a sensible JSON placeholder for this. Will add!